src/NeoSharp.BinarySerialization/BinaryDeserializer.cs

```diff
@@ -39,7 +38,7 @@ public static void RegisterTypes(params Assembly[] asms)
         /// <param name="data">Data</param>
         /// <param name="settings">Settings</param>
         /// <returns>Return byte array</returns>
-        public T Deserialize<T>(byte[] data, BinarySerializerSettings settings = null) where T : new()
+        public T Deserialize<T>(byte[] data, BinarySerializerSettings settings = null)
```

Comment:
Keep in my mind, Activator.CreateInstance is way more slower then new T()

Reply:
The problem is the arrays. I need to remove this if i want to allow Desearialize<Transaction[]> or similar